### PR TITLE
Docker Support

### PR DESCRIPTION
**Pull request by nfacha**
Adds docker/docker-compose support

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM node:16
```
Comment (bartosjiri) [inline Dockerfile]: Let's use some minimal image with smaller size, e.g. `node:lts-alpine`.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+version: '3.4'
+services:
+  lurker:
+    image: ghcr.io/nfacha/twitch-lurker-bot:master
```
Comment (bartosjiri) [inline docker-compose.yml]: We'll use locally built image, replacing `image` with:
```
build:
    context: "."
```

**Review by bartosjiri — CHANGES_REQUESTED** [review]
Hey @nfacha, thanks for the PR! 😍

I've added comments for some minor modifications. In general, I would prefer for users to build their image by themselves. You are still welcome to build and publish the image through your fork though!

Also, could you please update the README to let users understand how to use the script with Docker/Docker Compose? 

Thanks! 🎉

**Comment by nfacha** [comment]
> Hey @nfacha, thanks for the PR! heart_eyes
> 
> I've added comments for some minor modifications. In general, I would prefer for users to build their image by themselves. You are still welcome to build and publish the image through your fork though!
> 
> Also, could you please update the README to let users understand how to use the script with Docker/Docker Compose?
> 
> Thanks! tada

The changes have been made :)

**Pull request merged**